src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Controller/ReturnsControllerTests.cs

```diff
@@ -41,6 +41,11 @@ public void ReturnsControllerInheritsAatfReturnBaseController()
             typeof(ReturnsController).BaseType.Name.Should().Be(typeof(AatfReturnBaseController).Name);
         }
 
+        [Fact]
+        public void ReturnsController_ShouldHaveValidateOrganisationActionFilterAttribute()
+        {
```

Comment:
Empty unit test?

Reply:
Quite clearing losing the plot... added unit test